README.md

```diff
@@ -1,76 +1,75 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Calculator
 
-> One paragraph statement about the project.
+> A simple calculator app using React.
 
-![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+This project is part of the Microverse curriculum "React." 
 
-## Built With
+The calculator was made to practice the creation of a web application using React.
+The project is using classes instead of hooks.
 
-- Major languages
-- Frameworks
-- Technologies used
+Please, be mindful this project is under construction. 🛠
 
-## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+## Live Version
 
+[Live version here!](https://calculator-no4.herokuapp.com)
 
-## Getting Started
+## Prerequisites
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- [Node.js](https://nodejs.org/en/) installed.
+- Vs Code
 
+## Getting Started
 
-To get a local copy up and running follow these simple example steps.
+To get a local copy up and running, follow these simple example steps.
 
-### Prerequisites
+- At first, you have to clone this repository. Open your terminal and paste this without the $ sign.
 
-### Setup
+      $ git clone https://github.com/AlejandroNo4/calculator.git
 
-### Install
+- Now that you have the repository in your local machine. Change the direction of the local file with your terminal. Just type "$ cd" (without the $ sign) and the path to reach the calculator-app folder.<br/>
+  Something like:
 
-### Usage
+        $ cd User/Documents/calculator/calculator-app
 
-### Run tests
+- You are done with the installations! Now to run the project, first type:
 
```

Comment:
- [ ] [REQUIRED] It will be nice if you ask users to install packages by using `npm install` before` npm start`